### DISCUSSION TITLE
Hide read posts

### DISCUSSION THREAD

**Discussion by avidseeker** [mastodon/mastodon#31037] [Q&A]
I'm looking for a Mastodon client that supports hiding already seen posts.

* [Mastodon](https://github.com/mastodon/mastodon/issues/21193)
* [Moshidon](https://github.com/LucasGGamerM/moshidon/issues/437)
* [Mastodon-android](https://github.com/mastodon/mastodon-android/issues/70)
* [Tusky](https://github.com/tuskyapp/Tusky/issues/3961)
* [Pachli](https://github.com/pachli/pachli-android/issues/629)

Related:
* [Standardization draft to support this across Fediverse clients](https://socialhub.activitypub.rocks/t/4361)

**Comment by cweiske** [comment]
Related: https://github.com/mastodon/mastodon/issues/21193
